tests/python/release_checklist/check_lerobot_v3_dataloader.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import os
+from argparse import Namespace
+
+import rerun as rr
+from huggingface_hub import snapshot_download
+
+README = """\
+# LeRobot dataloader check
+
+This will load a small v3 LeRobot dataset -- simply make sure that it does.
+
+The LeRobot dataset loader works by creating a new _recording_ for each episode in the dataset.
+I.e., you should see exactly 3 recordings, corresponding to episode 0, 1 and 2.
+"""
+
+
+def run(args: Namespace) -> None:
+    # NOTE: The LeRobot dataloader works by creating a new recording for each episode.
+    # That means the `recording_id` needs to be set to "episode_0", otherwise the LeRobot dataloader
+    # will create a new recording for episode 0, instead of merging it into the existing recording.
+    # If you don't set it, you'll end up with 4 recordings, an empty one and the 3 episodes.
+    rec = rr.script_setup(args, f"{os.path.basename(__file__)}", recording_id="episode_0")
```

Comment:
This f string interpolation looks unnecessary. Nicer modern way with pathlib is also:
```suggestion
    rec = rr.script_setup(args, Path(__file__).name, recording_id="episode_0")
```